frontend/src/layout/navigation/TopBar/NotificationsBell.scss

```diff
@@ -0,0 +1,4 @@
+.NotificationsBell-Popup {
+    z-index: 949; // so TZ aware time pop-up is not behind this pop-up
```

Comment:
AFAIK we're keeping track of z-indexes in `vars.scss`

Reply:
https://github.com/PostHog/posthog/pull/12087